pennylane/_device.py

```diff
@@ -815,8 +815,8 @@ def supports_operation(self, operation):
             return operation.__name__ in self.operations
         if isinstance(operation, str):
 
-            if operation.endswith(Operation.string_for_inverse):
-                in_ops = operation[: -len(Operation.string_for_inverse)] in self.operations
+            if operation.endswith(".inv"):
+                in_ops = operation[:-4] in self.operations
```

Comment:
@albi3ro @mariaschuld let's think about even refactoring this. Ideally, calling `.inv` should set an internal boolean flag of the operation. It shouldn't mutate the name!